canvas_modules/common-canvas/src/common-properties/controls/checkboxset/checkboxset.jsx

```diff
@@ -74,27 +78,53 @@ class CheckboxsetControl extends React.Component {
 	}
 
 	render() {
+
+		const hidden = this.props.state === STATES.HIDDEN;
+		const disabledTip = this.props.state === STATES.DISABLED;
+
 		let controlValue = this.props.value;
 		if (typeof controlValue === "undefined" || controlValue === null) {
 			controlValue = [];
 		}
 		const checkboxes = [];
 		for (var i = 0; i < this.props.control.values.length; i++) {
+			let tooltip = "";
+			if (this.props.control.valueDescs && !this.props.tableControl) {
+				tooltip = (
+					<span >{this.props.control.valueDescs[i]}</span>
+				);
+			}
+			const tooltipIcon = isEmpty(this.props.control.valueDescs[i]) ? "" : (
```

Comment:
This probably should be null instead of "" otherwise we'll be adding "" into the DOM.

Reply:
Updated it.